project/backend/project-manager/docker-compose.yml

```
@@ -0,0 +1,43 @@
+      - DB_URL=jdbc:mysql://host.docker.internal:3306/metadocs
+      - DB_USERNAME=root
+      - DB_PASSWORD=1234
+    depends_on:
+      - otel-collector
+
+  otel-collector:
+    image: otel/opentelemetry-collector-contrib:0.123.0
+    container_name: otel-collector
+    volumes:
+      - ./collector-spring-native-config.yaml:/collector-spring-native-config.yaml
+    command: [ "--config=/collector-spring-native-config.yaml" ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)